galaxy/main/models.py

```diff
@@ -389,6 +389,22 @@ class Meta:
         null=True,
         verbose_name="Last Import"
     )
+    quality_score = models.FloatField(
```

Comment:
I just thought of one last thing. Don't forget to run `makemigrations` and commit the migration file for these changes.